src/Page/Index.elm

```diff
@@ -98,5 +98,61 @@ view _ _ _ =
             , Html.p []
                 [ Html.text "Gren produces small JavaScript files, and runs surprisingly fast. Applications written in Gren can be both smaller and faster than your average React-based application." ]
             ]
+        , Html.article
+            []
+            [ Html.h3 []
+                [ Html.text "Runs Anywhere" ]
+            , Html.pre []
+                [ Html.code [] 
+                    [ Html.text 
+                        """viewItems = Array String -> String
+viewItems items =
+    items
+        |> Array.sort
+        |> String.join ", "
+"""
+                    ]
+                ]
+            , Html.p []
+                [ Html.text "In the "
+                , Html.a 
+                    [ Attribute.href "https://packages.gren-lang.org/package/gren-lang/browser/version/latest/overview" ]
+                    [ Html.text "browser:" ]
+                ]
+            , Html.pre []
+                [ Html.code [] 
```

Comment:
Does it need to be `<pre>` followed by `<code>`? 🤔

Reply:
Good question.

I think it is pretty typical to use both in this case. `pre` being more of a layout element and `code` more for semantics about what is in the preformatted block.

MDN says "`<pre>` elements commonly contain `<code>`, `<samp>`, and `<kbd>` elements, to represent computer code, computer output, and user input, respectively."

Looking at the package site, looks like code samples there are rendered with `<pre><code class="language-elm">`

Reply:
Well, as I just discovered pre + code is also required for the styling to work as-is. So I'll be doing that even if the contents aren't code 😅